CS/CASESTUDY1/Bike.java

```diff
@@ -0,0 +1,41 @@
+public class Bike extends Vehicle {
+
+	private boolean selfStart;     		// bike is self start or not
+
+	private double helmetPrize;			// helmet will be gifted with bike or not
```

Comment:
helmetPrize is a element of type double but according to your comment it should be Boolean if it just contain "helmet will be gifted with bike or not"